src/modules/admin/components/formRules/NewFormRuleDialog.tsx

```diff
@@ -0,0 +1,304 @@
+                  removeTooltip={'Remove Condition'}
+                  sx={{
+                    backgroundColor: theme.palette.grey[100],
+                    border: 0,
+                    py: 1,
+                  }}
+                >
+                  <Grid container spacing={1}>
+                    <Grid item xs={2} sm={1}>
+                      <FormRuleLabelTypography>
+                        {index === 0 ? 'If' : 'And'}
+                      </FormRuleLabelTypography>
+                    </Grid>
+                    <Grid item xs={9} sm={4}>
+                      <FormSelect
+                        disableClearable={true}
+                        value={{ code: conditionType }}
+                        options={conditionPickList}
+                        onChange={(_event, option) => {
+                          if (isPickListOption(option)) {
+                            const newCondition = option.code as ConditionType;
+                            setRule({
+                              ...rule,
+                              [conditionType]: undefined,
+                              [option.code]: pickListMap[newCondition][0].code,
+                            });
+                          }
+                        }}
+                      />
+                    </Grid>
+                    <Grid item xs={2} sm={1}>
+                      <FormRuleLabelTypography
+                        sx={isTiny ? {} : { textAlign: 'center' }}
+                      >
+                        is
+                      </FormRuleLabelTypography>
+                    </Grid>
+                    <Grid item xs={10} sm={5}>
+                      <FormSelect
+                        // disable clearing the input because we expect users to remove the whole condition
+                        disableClearable={true}
+                        value={value ? { code: value } : undefined}
+                        options={pickListMap[conditionType]}
+                        onChange={(_event, option) => {
+                          if (isPickListOption(option)) {
+                            setRule({ ...rule, [conditionType]: option.code });
+                          }
+                        }}
+                      />
+                    </Grid>
+                  </Grid>
+                </RemovableCard>
+              );
+            })}
+
+          {rule.funder === FundingSource.LocalOrOtherFundingSource && (
+            <Card
+              sx={{
+                backgroundColor: theme.palette.grey[100],
+                border: 0,
+                px: 2,
+                py: 1,
+              }}
+            >
+              <Grid container spacing={1}>
+                <Grid item xs={4}>
+                  <FormRuleLabelTypography>
+                    And funding source is
+                  </FormRuleLabelTypography>
+                </Grid>
+                <Grid item xs={8}>
+                  <TextInput
```

Comment:
It would be great for this to be a picklist of distinct values from the `OtherFunder` column for all HMIS Projects. The string needs to match exactly, so it's more error-prone to leave it up to the user. Would you mind creating a follow-up ticket for that? Or, you're welcome to bring it into this work instead if you'd like. It will require another backend change to add a new remote pick list (perhaps called `LOCAL_OR_OTHER_FUNDERS`).

Reply:
Sounds good, I created https://github.com/open-path/Green-River/issues/6317

Reply:
Thanks!